### PR TITLE
BUG: In IE11 hls stopped after first segment

### PR DESCRIPTION
**Pull request by sea-kg**
## Description
BUG: In IE11 hls stopped after first segment

## Specific Changes proposed
Please list the specific changes involved in this pull request.

### CODE REVIEW

**Comment by gesinger** [comment]
Hey @sea-kg , have you tried using the `withCredentials` option? https://github.com/videojs/videojs-contrib-hls#withcredentials

File: src/xhr.js [hunk 1]
```diff
@@ -17,6 +17,10 @@ import {
 
 const xhrFactory = function() {
   const xhr = function XhrFunction(options, callback) {
+    
+    // allow use cookie
+    options.withCredentials = true;
```
Comment (gesinger) [inline src/xhr.js]: This is available via an option: https://github.com/videojs/videojs-contrib-hls#withcredentials

Comment (gesinger) [inline src/xhr.js]: This is available through options: https://github.com/videojs/videojs-contrib-hls#withcredentials

**Comment by forbesjo** [comment]
Closing this since video.js should shim the vttcue automatically and the original issue is fixed by using the `withCredentials` option

**Pull request closed without merging**